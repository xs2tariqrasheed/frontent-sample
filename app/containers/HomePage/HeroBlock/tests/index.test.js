import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';
import { Provider } from 'react-redux';
import { browserHistory, BrowserRouter } from 'react-router-dom';
import configureStore from '../../../../configureStore';

import Hero from '../index';

describe('<Hero />', () => {
  let store;

  beforeAll(() => {
    store = configureStore({}, browserHistory);
  });

  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <Provider store={store.store}>
        <BrowserRouter>
          <IntlProvider locale="en">
            <Hero />
          </IntlProvider>
        </BrowserRouter>
      </Provider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
