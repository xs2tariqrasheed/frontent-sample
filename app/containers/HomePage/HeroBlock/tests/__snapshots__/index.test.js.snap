// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hero /> should render and match the snapshot 1`] = `
<div
  class="bg-green-golf-green"
>
  <div
    class="relative z-10 bg-transparent"
  >
    <iframe
      allow="autoplay; fullscreen"
      allowfullscreen=""
      class="w-full"
      frameborder="0"
      height="360"
      src="https://player.vimeo.com/video/454774621"
      title="trailer"
      width="640"
    />
    <div
      class="gridcontainer grid-flow-row grid-flow-col text-center md:mt-5"
    >
      <a
        class="btn inline-block no-underline text-center md:w-52 mr-5 mb-5"
        href="/playtoearn/training"
      >
        Learn About Training
      </a>
      <a
        class="btn inline-block no-underline text-center md:w-52 mr-5 mb-5"
        href="/playtoearn/proshop"
      >
        Learn About Pro Gear
      </a>
    </div>
  </div>
</div>
`;
